docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,367 @@
+
+Here you should have a list of three keys entries: 
+
+* EncryptionTemplate,
+* GeneralPurposeTemplate,
+* SignatureTemplate.
+
+Replace each value with our newly created template. The default value should be `IPSECIntermediateOffline`.
+
+You now need to reboot the NDES server to apply changes to the registry.
+
+IIS configuration
+^^^^^^^^^^^^^^^^^
+
+The use of SCEP with PacketFence also require a change in the IIS configuration.
+
+Navigate to `Server Manager` > `Web(IIS)` > expand `Default web site` > select `CertSrv` > `mscep`. Select `Authentication`, and double click `Anonymous Authentication`. Make sure that `Application pool identity` is selected.
+
+Online Certificate Status Protocol (OCSP)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For the configuration of OCSP, there are small changes to apply.
+
+First we need to allow the use of the template "OCSPResponseSigning" by the server, to do so navigate to `Server Manager` > `Roles` > `Active Directory Certificates Services` > expand `YourServerName`, right click `Certificate template` and chose `New template to issue`, in the list select "OCSPResponseSigning".
+
+After the installation of OCSP we need to create a "Revocation Configuration".
+
+To create the "Revocation Configuration" navigate to `Server Manager` > `Roles` > `Active Directory Certificate Services` and expand `OnlineResponder`. Then expand `YourServerName`, right click on "Revocation Configuration", select `Add Revocation Configuration`, click `Next`, chose a name for your configuration and click `Next`. 
+
+Chose `Select a certificate for an existing entreprise CA`, click `Next`. Click `Browse` and find your entreprise CA in the list, select it, click `OK` and then `Next`. Chose `Automatically select a signing certificate`, make sure `Auto-Enroll for an OCSP signing certificate` is selected, then chose the default template of OCSP which is `OCSPResponseSigning` in the drop down list next to `Certificate Template:`. You need to add providers only if you whish to use some CRL in addition to OCSP.
+
+Once created, right click the revocation configuration and select `Edit properties`, go to the tab `Signing`, here you need to select `Enable NONCE extension support` then click `OK`.
+
+Verify that your OCSP is in the settings of the CA. Right click your CA, chose `Properties`. Navigate to the tab `Extension`, in the drop down list `Select extension` chose `Authority Information Access (AIA)`. Make sure that you have the following in the list of locations: `http://<fqdn of ndes server>/ocsp`, if you do not have it, add it via the button `Add...`. In this menu type the "http://" then insert "<ServerDNSName>" and type "/ocsp", the line you add should look like the following: `http://<ServerDNSName>/ocsp`, then validate by clicking `OK`. Also verify that `Include in the online certificate status protocol(OCSP) extension` is selected.
+
```

Comment:
Based on the test i did we probably have to change ocsp config to be able to have a sort of real time ocsp (by default if you revoke a certificate it can take 2 days until ocsp will be up to date)